### DISCUSSION TITLE
🐙 v3.8.x

### DISCUSSION THREAD

**Discussion by trexfeathers** [SciTools/iris#5363] [Releases]
<div align="center">

| Manager | Buddy | Feature Branch | Project |
| :---: | :---: | :---: | :---: |
| @stephenworsley  | @pp-mo  | [v3.8.x](https://github.com/SciTools/iris/tree/v3.8.x) | [🐙Iris v3.8.0](https://github.com/orgs/SciTools/projects/26) |

</div>

<div align="center">

| Minor Release | Status | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.8.0rc0` | [![status](https://img.shields.io/badge/v3.8.0rc0-released-green)](https://github.com/SciTools/iris/discussions/5363) | `21-02-2024` | [Docs](https://scitools-iris.readthedocs.io/en/v3.8.0rc0/index.html) | [v3.8.0rc0](https://github.com/SciTools/iris/releases/tag/v3.8.0rc0) | [rc_iris](https://anaconda.org/conda-forge/iris?version=3.8.0rc0&channel=rc_iris) | [scitools-iris (--pre)](https://pypi.org/project/scitools-iris/3.8.0rc0/) |
| `v3.8.0` | [![status](https://img.shields.io/badge/v3.8.0-released-green)](https://github.com/SciTools/iris/discussions/5363) | `29-02-2024` | [Docs](https://scitools-iris.readthedocs.io/en/v3.8.0/index.html) | [v3.8.0](https://github.com/SciTools/iris/releases/tag/v3.8.0) | [main](https://anaconda.org/conda-forge/iris?version=3.8.0&channel=main) | [scitools-iris](https://pypi.org/project/scitools-iris/3.8.0/) |
| `v3.8.1` | [![status](https://img.shields.io/badge/v3.8.1-released-green)](https://github.com/SciTools/iris/discussions/5363) | `04-03-2024` | [Docs](https://scitools-iris.readthedocs.io/en/v3.8.1/index.html) | [v3.8.1](https://github.com/SciTools/iris/releases/tag/v3.8.1) | [main](https://anaconda.org/conda-forge/iris?version=3.8.1&channel=main) | [scitools-iris](https://pypi.org/project/scitools-iris/3.8.1/) |

</div>

**Comment by rcomer** [comment]
Am I reading this right that the next minor release will be in 2024?  Asking because I have a branch that requires bumping the numpy min pin to 1.23 (which, according to [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html) we cannot do till January).

**Comment by stephenworsley** [comment]
The release has been pushed up from February 22nd to February ~~31st~~ _29th_ to fully accommodate the planned work in this release.

**Reply by rcomer** [reply]
Do you mean February 29th?  Or March 31st?

**Reply by stephenworsley** [reply]
oops, I'd gotten mixed up with the calendar there, it should be the 29th.

**Comment by trexfeathers** [comment]
Release candidate delayed until `2024-02-21`, apologies for any inconvenience.
